users/views.py

```diff
@@ -98,7 +98,7 @@ def get_calandar_days(year=None, month=None):
         This list will include dates from the month before if the current month hasn't started with sunday """
 
     current_week = list()
-    calendar = Calendar()
+    calendar = cal
```

Comment:
please use cal no need to define one more variable 
```suggestion
```

Reply:
done